apps/web/src/pages/TalentNominations/NominateTalent/components/Details.tsx

```diff
@@ -675,14 +676,25 @@ const Details = ({ detailsQuery, optionsQuery }: DetailsProps) => {
     nominationOptions = [...nominationOptions, "developmentProgram"];
   }
 
+  const referenceSet =
+    !!talentNomination.advancementReference?.id ||
+    !!talentNomination.advancementReferenceFallbackName;
+
+  let defaultReference: Maybe<string> | undefined;
+
+  if (referenceSet) {
+    defaultReference = talentNomination.advancementReference?.id
+      ? talentNomination.advancementReference.id
+      : null;
```

Comment:
This could just be 
`= talentNomination.advancementReference?.id ?? null`
right?